web/util/date.ts

```diff
@@ -3,7 +3,7 @@ import { addMinutes, intervalToDuration } from 'date-fns';
 import enUS from 'date-fns/locale/en-US';
 
 export const formatTime = (date: Date, timezone: string, meridiem?: boolean) =>
-  formatInTimeZone(date, timezone, meridiem ? 'h:mm aa' : 'HH:mm', {
+  formatInTimeZone(date, timezone, `h:mm${meridiem ? ' aa' : ''}`, {
```

Comment:
I think the confusion that arose regarding `meridiem` suggests that the parameter name is too vague. I think it would be good to rename it and/or to add function documentation[1].

Second, given that it is now clear that the parameter isn't a toggle between 12-hour and 24-hour anymore (a change that makes sense), I really think the default should be to _show_ the AM/PM marker, so that the caller must explicitly opt in to the ambiguous rendering.

Combing the two comments above, I would suggest replacing `meridiem?` with something like `hideMeridiemIndicator?`, inverting the logic.

[1] We have few of those now, but for programs of a certain size/complexity I really prefer to be able to read some JSdoc or equivalent, rather than having to read implementation details several levels deep in order to try to piece together some implied contract.

Reply:
2e239a8.